docs/source/index.md

```diff
@@ -1,34 +1,93 @@
-# Welcome to SimPhoNy docs!
-Here you can browse though the general documentation of SimPhoNy.
+# Welcome to the SimPhoNy docs!
 
-**Compatibility matrix**
+SimPhoNy is an ontology-based open-source Python-framework that promotes and enables interoperability between any 3rd-party software tool. Here you can learn more about it.
 
 ```eval_rst
- =============  ==========
- SimPhoNy docs  OSP-core  
- =============  ==========
- 2.3.x          3.4.0-beta
- 2.2.x          3.3.5-beta
- 2.1.x          3.3.0-beta
- =============  ==========
-```
 
-```eval_rst
+.. panels::
+   :body: text-center
+
+   ---
+   **Getting Started**
+
+   Overview, main concepts, and installation guide
+
+   .. link-button:: overview.html
+      :text: To the getting started guides
+      :classes: btn-outline-primary stretched-link
+
+   ---
+   **Usage Guide**
+
+   Core functionalities, wrappers and advanced utilities
+
+   .. link-button:: jupyter/cuds-api.ipynb
+      :text: To usage guide
```

Comment:
Some buttons say _to ..._, and others _to the ..._. Might be better to unify the phrasing. At least add _the_ to this one?